packages/transport-grpc/src/send-get-transaction-status.js

```diff
@@ -8,16 +8,28 @@ const STATUS_MAP = {
   2: "FINALIZED",
   3: "EXECUTED",
   4: "SEALED",
-  5: "EXPIRED"
+  5: "EXPIRED",
 }
 
-const u8ToHex = (u8, context) => context.Buffer.from(u8).toString("hex")
+const u8ToHex = (u8, context) =>
+  !u8.reduce((empty, b) => empty && !b, true)
```

Comment:
Could you explain this logic here? I think we can simply use the `u8ToHex` function without modification. Was there an issue with the function that you ran into?

Reply:
When the transaction has not been executed yet, there is no assigned blockId.  In the HTTP API, this is null, however with GRPC it is just a buffer of null bytes (i.e. all 0s).  I basically just added a check whether the u8 buffer is null bytes and return null if this is the case in order to maintain parity with the HTTP API

Reply:
Ok makes sense - could you rename the function to something that make this obvious? Perhaps `nonNullU8ToHex`, or whatever you prefer